pkg/bbgo/order_executor_general.go

```diff
@@ -356,11 +356,11 @@ func (e *GeneralOrderExecutor) OpenPosition(ctx context.Context, options OpenPos
 }
 
 // GracefulCancelActiveOrderBook cancels the orders from the active orderbook.
-func (e *GeneralOrderExecutor) GracefulCancelActiveOrderBook(ctx context.Context, activeOrders *ActiveOrderBook) error {
+func (e *GeneralOrderExecutor) GracefulCancelActiveOrderBook(ctx context.Context, activeOrders *ActiveOrderBook, orders ...types.Order) error {
```

Comment:
The purpose of this method `GracefulCancelActiveOrderBook` is to cancel the orders from the given ActiveOrderBook, so we should avoid adding extra order parameters.